### ISSUE TITLE
Arrival times are too generic

### ISSUE THREAD

**Issue by line72** [line72/montclair#53]
I like that we use moment to say "in 9 minutes" and so forth, but anything passed an hour is pretty useless, since it just rounds to the closest hour. I think if the arrival time is > 1 hour, then we should just use the exact time it will arrive: 16:19 or 4:19pm depending on locale

**Comment by line72** [comment]
We should be able to do:
```
const arrival = moment().diff(a.arrival) > 1000 * 60 * 60 ? a.format('LT') ? a.fromNow();
```

**Comment by line72** [comment]
Merged in master in da88822c3d37aba9550fade5b2709e64887e9595
